test/test_train_mp_imagenet.py

```diff
@@ -223,7 +228,6 @@ def train_loop_fn(loader, epoch):
         xm.all_reduce('sum', gradients, scale=1.0 / xm.xrt_world_size())
         scaler.step(optimizer)
         scaler.update()
-        xm.mark_step()
```

Comment:
I found this `mark_step` is redundant since the script is using `pl.MpDeviceLoader` and it diminishes the performance gain of the `zero_gard` optimization.